src/components/assessment/assessmentShape.ts

```diff
@@ -149,3 +150,18 @@ export type Library = {
     2?: string; // For mission control
   }>;
 };
+
+export type AutogradingResult = {
+  resultType: string;
+  expected?: string;
+  actual?: string;
```

Comment:
same for this

Reply:
I added comments to clarify what expected and actual means. I won't be changing this because the type follows the JSON format coming from the backend.

Reply:
Okay this is fine